pkg/webhook/example_test.go

```diff
@@ -70,3 +80,72 @@ func Example() {
 		panic(err)
 	}
 }
+
+// This example creates a webhook server that can be
+// ran without a controller manager.
+func ExampleServer_StartStandalone() {
+	// Create a webhook server
+	hookServer := &Server{
+		Port: 8443,
+	}
+
+	// Register the webhooks in the server.
+	hookServer.Register("/mutating", mutatingHook)
+	hookServer.Register("/validating", validatingHook)
+
+	// Start the server without a manger
+	err := hookServer.StartStandalone(signals.SetupSignalHandler(), scheme.Scheme)
```

Comment:
@kevindelgado one more thing - maybe we could write an e2e for examples to help catch these, but it looks like the cert and key are not optional on this server? When using code similar to this, I get:

```
time="2021-04-23T20:34:54Z" level=error msg="Failed to serve admission webhooks." error="open /tmp/k8s-webhook-server/serving-certs/tls.crt: no such file or directory"
```

Reply:
that's intentional -- same with the main webhook server.  More broadly, the main point of the webhook server is to help with managing TLS setup properly.  If you don't want TLS, use StandaloneWebhook.

We could certainly document that better, though.

Reply:
https://github.com/kubernetes-sigs/controller-runtime/pull/1501